pytorch/mnist/requirements.txt

```diff
@@ -2,6 +2,6 @@ numpy==1.18.1
 pandas==1.0.1
 pillow==6.2.0
 tenacity==5.1.1
-tensorboardX==1.4
+tensorflow==1.15.2
```

Comment:
Do we need tensorflow installed to access torch's implementation of tensorboard?

Reply:
Yes. For example, `tensorboard/summary/writer/event_file_writer.py` imports tensorflow. I ran into this error when I was only installing tensorboard.